src/game_events/pump.cpp

```diff
@@ -536,30 +554,39 @@ bool t_pump::operator()()
 				++impl_->internal_wml_tracking;
 			}
 		}
-		// Initialize an iteration over event handlers matching this event.
+
```

Comment:
What's with the changes here? Are they relevant to the PR, or just extra cleanup? If the latter, they should be in a separate PR. (The Boost->Std changes that I noticed in several files would be better separated too, but that's more for future reference; I don't consider it important enough to insist you change them.)